lib/chef/resource/build_essential.rb

```diff
@@ -22,7 +22,12 @@ class BuildEssential < Chef::Resource
       resource_name :build_essential
       provides :build_essential
 
-      property :compile_time, [true, false], default: false
+      description "Use the build_essential resource to install packages required for compiling C software from source"
+      introduced "14.0"
+
+      property :compile_time, [true, false],
```

Comment:
Shouldn't the type (and default) here be [TrueClass, FalseClass] for consistency?